docs/de/docs/deployment/index.md

```diff
@@ -0,0 +1,7 @@
+# Deployment - Einführung
+
+Das Bereitstellen einer **FastAPI** Anwendung ist relativ einfach.
+
+Hierbei gibt es abhängig von der Situation und den eingesetzen Technologien mehrere Möglichkeiten.
```

Comment:
Maybe `Werkzeuge` ?

Reply:
Isn´t `Werkzeuge` in this context to specific?

Reply:
For me, deployment is often about continuous deployment, so I'm thinking more along the tools spectrum. However, I have taken a closer look at the next chapters. It's less about the tools used, but more about the techniques behind them. Literally, "Werkzeuge" are more appropriate, but here it's more about the context. That's why I think "eingesetzen Technologien" is more appropriate.